core-bundle/src/Doctrine/Schema/DcaSchemaProvider.php

```diff
@@ -410,22 +410,20 @@ private function getDefaultIndexLength(Table $table): int
 
         $largePrefix = $this->doctrine
             ->getConnection()
-            ->query("SHOW VARIABLES LIKE 'innodb_large_prefix'")
-            ->fetch(\PDO::FETCH_OBJ)
+            ->fetchAssociative("SHOW VARIABLES LIKE 'innodb_large_prefix'")
         ;
 
         // The variable no longer exists as of MySQL 8 and MariaDB 10.3
-        if (false === $largePrefix || '' === $largePrefix->Value) {
+        if (false === $largePrefix || '' === $largePrefix['Value']) {
             return 3072;
         }
 
         $version = $this->doctrine
             ->getConnection()
-            ->query('SELECT @@version as Value')
-            ->fetch(\PDO::FETCH_OBJ)
+            ->fetchAssociative('SELECT @@version as Value')
```

Comment:
same here  (see https://github.com/contao/contao/pull/2421/files#r504547950) with `version`

Reply:
Changed in f11adc4e99d984b1c13aee23870e08136009c4c9.